lib/mayaUsd/utils/loadRulesAttribute.cpp

```diff
@@ -33,7 +33,7 @@ bool hasLoadRulesAttribute(const MFnDependencyNode& depNode)
 {
     MString nodeName = depNode.absoluteName();
     MString cmd;
-    cmd.format("addAttr -query -exists \"^2s.^1s\"", loadRulesAttrName, nodeName);
+    cmd.format("attributeQuery -exists -n \"^2s\" \"^1s\"", loadRulesAttrName, nodeName);
```

Comment:
Looks good. FYI, you don't really lines 39/40 as the default value for both these args is false.